docs/cue/reference/remap/functions/assert_eq.cue

```diff
@@ -0,0 +1,59 @@
+package metadata
```

Comment:
Could we add the same `notice` we have on `assert` here?

Reply:
Ah, derp. Totally meant to do that. Fixed.